### DISCUSSION TITLE
GraphQL API CreateIssue doesn't work with projectIds parameter?

### DISCUSSION THREAD

**Discussion by marcfishman** [community/community#62101] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I've been trying to create issues using the GraphQL `CreateIssue` API, which works as expected. However, when I try to create an issue in a specific project using the `projectIds` parameter, it fails, saying that the node ID that I put in the payload is not found.

The Node ID that I am using for the project is the one that I got from the `query -> organization -> projectsV2` GraphQL API, so I know that it's the right ID and that it exists.

I've found a workaround by making 2 separate calls:
1. CreateIssue, and then
2. AddProjectV2ItemById, using the ID of the issue created in step 1 as the `contentId` for this call, and then passing the same `projectId`. 
(This also proves that the ID definitely exists).

I'm wondering if this is some sort of permissions issue, or if it is possibly a bug in the API?

Thanks!

**Comment by ndamulelonemakh** [comment]
Still experiencing this in 2024

**Comment by sartizalamayon** [comment]
Based on your description, the Node ID for the project exists and can be used in separate API calls, but the error suggests the Node ID is not being recognized during issue creation. I think the workaround you found is valid. You can continue to use it until GitHub resolves any potential bugs in the API.
